### PR TITLE
Add googletest as unit test

### PR DESCRIPTION
**Pull request by jaebaek**
RELATED ISSUE: #25

### CODE REVIEW

**Comment by jaebaek** [comment]
This commit simply added googletest.
Currently many functions in [vkrunner/](vkrunner) are static functions and the unit test cannot link them. To test those static functions, we need wrappers that functions in unit tests can access.

File: unittest/script.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+#include <iostream>
+#include <sstream>
+#include <stdlib.h>
+#include <string>
+
+#include "gtest/gtest.h"
+
+extern "C" {
+#include "vr-config.h"
```
Comment (samuelig) [inline unittest/script.cpp:31]:
```suggestion
#include "vr-config-private.h"
```

File: unittest/script.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#include "vr-list.h"
+#include "vr-script.h"
```
Comment (samuelig) [inline unittest/script.cpp:33]:
```suggestion
#include "vr-script-private.h"
```

**Comment by samuelig** [comment]
Repository moved to https://gitlab.freedesktop.org/mesa/vkrunner. Please re-submit this pull request there.

**Pull request closed without merging**